recipe/patches/0001-Fixes-tests-to-be-compatible-with-Python-3.13.patch

```diff
@@ -0,0 +1,165 @@
+From b8ca4f3808c5fdd60a94dfd16022e7fc773766b7 Mon Sep 17 00:00:00 2001
+From: Jaida Rice <jaidarice@gmail.com>
+Date: Tue, 14 Jan 2025 14:07:11 -0800
+Subject: [PATCH] Fixes tests to be compatible with Python 3.13
+
+---
+ boltons/ioutils.py   |  4 ++--
```

Comment:
It's not modifying the tests, but the actual code of boltons. We should not go that route.